libs/logging/src/base_types/log_source.ts

```diff
@@ -1,14 +1,19 @@
 export enum LogSource {
   System = 'system',
   VxAdminFrontend = 'vx-admin-frontend',
+  VxAdminFrontendServer = 'vx-admin-frontend-server',
```

Comment:
A little pedantic to make this a new set of names but I suppose its more accurate

Reply:
Yeah, noticed that the `prodserver` logs weren't getting logged - we have logic to skip logging for the frontend source IDs if there's no kiosk browser - so, considered switching to using the backend source IDs, but thought that could be confusing in the logs